examples/runqslower/src/main.rs

```diff
@@ -70,37 +69,31 @@ fn handle_lost_events(cpu: i32, count: u64) {
 fn main() -> Result<()> {
     let opts = Command::from_args();
 
-    if !opts.obj_path.as_path().exists() {
-        eprintln!("{} does not exist", opts.obj_path.as_path().display());
-        exit(1);
-    }
-
-    let mut obj_builder = ObjectBuilder::default();
+    let mut obj_builder = RunqslowerObjectBuilder::default();
```

Comment:
this is not really Object, though, right? It's the skeleton. I'd go with using "Skel" instead of "Object" everywhere.

Reply:
Yeah, that makes sense.